packages/components/badge/src/getBadgeStyles.ts

```diff
@@ -1,5 +1,5 @@
 import tokens from '@contentful/f36-tokens';
-import { Interpolation } from '@emotion/core';
+import type { Interpolation } from '@emotion/serialize';
```

Comment:
Should this be added as an explicit dependency?

Reply:
good point, I decided to remove explicit type here